### DISCUSSION TITLE
How to display a user by SQL Queries

### DISCUSSION THREAD

**Discussion by hyzx86** [OrchardCMS/OrchardCore#15843] [Q&A]
I was create  a user extension module .
Use sql query and  liquid to display custom property on page. 

SQL Query（with out "return document" option）

```
select Content FROM Document d
inner join   UserIndex u on d.Id=u.DocumentId 
where d.Type='OrchardCore.Users.Models.User, OrchardCore.Users.Core'

```

Liquid
```
{% assign users = Queries.AllUsers | query %}
<ul>
{% for item in users %}
 <li>{{  item.Content   }}</li>
{% endfor %}
 </ul>
```
Page response:

`{"Id":61,"UserName":"admin","NormalizedUserName":"ADMIN","Email":"xx@xx.com","NormalizedEmail":"xx@xx.COM","PasswordHash":"xxxx==","SecurityStamp":"xxxx","EmailConfirmed":false,"ResetToken":null,"RoleNames":["Administrator"],"LoginInfos":[],"Properties":{"JZUserInfo":{"RealName":"张三"}}}`

How to only display RealName field.

Thanks.

**Comment by sebastienros** [comment]
`{{ item.Content.Properties.JZUserInfo.RealName }}`

**Comment by hyzx86** [comment]
Thanks ，but  there is nothing to display..

![image](https://user-images.githubusercontent.com/15613121/47195101-fab75080-d38c-11e8-8a28-c4847be7fa2d.png)
![image](https://user-images.githubusercontent.com/15613121/47195228-8df08600-d38d-11e8-889f-379369e8a65b.png)

**Comment by sebastienros** [comment]
It might be that the `Properties` parameter is not allowed in liquid. I will test that.

**Comment by hyzx86** [comment]
@sebastienros 
Thanks, 
Here is the source soce of the custom module
https://github.com/hyzx86/OrchardCore/blob/142f9bc986b3bc19f837e9e822272f74ebab4e9c/UserExtInfoSrc/JZUserExtensions.OrchardCore/Service/JZUserInfoService.cs#L75
